ignition/aws/master.yaml.tmpl

```diff
@@ -0,0 +1,2070 @@
+storage:
+  files:
+    - path: /srv/calico-kube-controllers-sa.yaml
+      filesystem: root
+      mode: 0644
+      contents:
+        inline: |
+          apiVersion: v1
+          kind: ServiceAccount
+          metadata:
+            name: calico-kube-controllers
+            namespace: kube-system
+    - path: /srv/calico-node-sa.yaml
+      filesystem: root
+      mode: 0644
+      contents:
+        inline: |
```

Comment:
interesting, i have not seen `inline` in the default ignition 
i guess its part of `ct`

Reply:
i'm using ct. No other way to manage "raw" json ignition. 

Look here https://github.com/giantswarm/giantnetes-terraform/blob/dev/platforms/aws/giantnetes/main.tf#L71

Unfortunately we need to get ct_config provider manually, because it's not in official TF repos. :(

Reply:
that sad, but atleast its working
and when we will have CD we dont have to care that much as it  will be part of the CD config